cob4_homing/scripts/calib.py

```diff
@@ -16,10 +16,18 @@
 WHEEL2 =  'b_caster_rotation_joint'
 WHEEL3 = 'fr_caster_rotation_joint'
 
+# old FDMs
+#tests = {
+#    ('please align wheels 1 and 2',  30000, (WHEEL1, -30000), (WHEEL2, -150000)),
+#    ('please align wheels 2 and 3', 150000, (WHEEL2, -30000), (WHEEL3, -150000)),
+#    ('please align wheels 3 and 1', -90000, (WHEEL3, -30000), (WHEEL1, -150000)),
+#}
+
+# new FDMs
 tests = {
-    ('please align wheels 1 and 2',  30000, (WHEEL1, -30000), (WHEEL2, -150000)),
-    ('please align wheels 2 and 3', 150000, (WHEEL2, -30000), (WHEEL3, -150000)),
-    ('please align wheels 3 and 1', -90000, (WHEEL3, -30000), (WHEEL1, -150000)),
+    ('please align wheels 1 and 2', -150000, (WHEEL1, -120000), (WHEEL2, 120000)),
+    ('please align wheels 2 and 3',  -30000, (WHEEL2, -120000), (WHEEL3, 135000)),
```

Comment:
Your global angles are fine, they actually make more sense as they correspond to the vectors.
However I am not sure about the `135000`.